Wikipedia/Code/WMFBarButtonItemPopoverMessageViewController.h

```diff
@@ -1,10 +1,13 @@
 @import UIKit;
 @import WMF.Swift;
 
-@interface WMFBarButtonItemPopoverMessageViewController : UIViewController <UIPopoverPresentationControllerDelegate, WMFThemeable>
+@interface WMFBarButtonItemPopoverMessageViewController : UIViewController <UIPopoverPresentationControllerDelegate>
 
```

Comment:
I'm getting a crash with this message `Thread 1: Could not find a storyboard named WMFBarButtonItemPopoverMessageViewController' in bundle NSBundle` maybe when the popover tries to instantiate.
I did a fresh install of the app, logged out (on simulator)

Reply:
@mazevedofs Good catch 😅 I think I have fixed this, let me know if it's working for you.